src/radical/pilot/configs/resource_ornl.json

```diff
@@ -163,9 +169,12 @@
         "system_architecture"         : {
                                          "smt"           : 2,
                                        # "options"       : ["nvme"],
-                                         "blocked_cores" : [ 0,  8, 16, 24, 32,  40,  48,  56,
-                                                            64, 72, 80, 88, 96, 104, 112, 120],
-                                         "blocked_gpus"  : []
+                                         "blocked_cores" : [ 0,   8,  16,  24,
+                                                            32,  40,  48,  56,
+                                                            64,  72,  80,  88,
+                                                            96, 104, 112, 120],
+                                         "blocked_gpus"  : [],
+					 "n_partitions"  : 8
```

Comment:
for default value we should keep `"n_partitions" : 1`

Reply:
`n_partitions` and `SMT=1` are last changes for this PR, and it is ready

p.s. I've added a corresponding comment in RU (branch `fix/flux_log`), so if you would want to follow it, then the following one should be in sync
https://github.com/radical-cybertools/radical.pilot/blob/ce2402bca95675b0f45aeb5612d8a435064dbca1/src/radical/pilot/agent/launch_method/flux.py#L101